semgrep-core/src/core/ast/AST_generic.ml

```diff
@@ -897,7 +897,11 @@ and stmt = {
      and before matching.
   *)
   (* used in semgrep to skip some AST matching *)
-  mutable s_bf : Bloom_filter.t option; [@equal fun _a _b -> true] [@hash.ignore]
+  mutable s_bf : Bloom_filter.t option;
+      [@equal fun _a _b -> true] [@hash.ignore]
+  mutable s_range :
+    (Parse_info.token_location * Parse_info.token_location) option;
+      [@hash.ignore]
```

Comment:
If this field is related to bloom-filter optimization, it should clearly indicate so. A comment + maybe a name containing `bf` would help.

Reply:
It isn't; `s_range` is just saving the range of the statement